### PR TITLE
Fix the Wear Session Details view.

### PR DESCRIPTION
**Pull request by yschimke**
Strip bad characters, split on paragraphs, and fix layout in the ScalingLazyColumn.

[text.webm](https://user-images.githubusercontent.com/231923/212468655-d2271576-14f5-46ba-90e4-828cb33d2907.webm)

### CODE REVIEW

File: wearApp/src/main/java/dev/johnoreilly/confetti/wear/sessiondetails/SessionDetailsView.kt [hunk 1]
```diff
@@ -30,59 +32,91 @@ import java.time.ZoneOffset
...
-                    }
-                )
+                SessionSpeakerInfo(speaker = speaker.speakerDetails)
             }
         }
     }
 }
 
-@RequiresApi(Build.VERSION_CODES.O)
+private fun SessionDetails?.descriptionParagraphs(): List<String> {
```
Comment (yschimke) [inline wearApp/src/main/java/dev/johnoreilly/confetti/wear/sessiondetails/SessionDetailsView.kt:85]: @joreilly any thoughts on how to handle this? During the load?

Comment (joreilly) [reply wearApp/src/main/java/dev/johnoreilly/confetti/wear/sessiondetails/SessionDetailsView.kt:85]: @martinbonnin I think you were hoping to "santize" this data at some point? (this is specific I think to fosdem info?)

Comment (martinbonnin) [reply wearApp/src/main/java/dev/johnoreilly/confetti/wear/sessiondetails/SessionDetailsView.kt:85]: Indeed, this is only for Fosdem data that is HTML. I can move that logic to the backend

Comment (yschimke) [reply wearApp/src/main/java/dev/johnoreilly/confetti/wear/sessiondetails/SessionDetailsView.kt]: https://github.com/joreilly/Confetti/issues/199

**Pull request merged**